package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "edx-ora2",
-  "version": "2.13.9",
+  "version": "3.0.2",
```

Comment:
@mattcarter: FYI: Should there be a comment or PR template item for keeping this in sync with the setup.py version?